app/src/main/java/com/xmartlabs/scasas/criminalintent/ui/crime/list/CrimeListFragment.java

```diff
@@ -77,7 +77,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
     switch (item.getItemId()) {
       case R.id.menu_item_new_crime:
         Crime crime = Crime.builder()
-            .id(generateUniqueIdentifier())
+            .id(generateUniqueIdentifier().toString())
```

Comment:
In my Opinion it's not the best solution because if you are using a `UUID` you should use this class. I mean, is like you define the `solved` field as the boolean string representation. I know, I'm being dramatically
I think there are two options:
1 Use a database converter to save the UUID as a string
```java
@com.raizlabs.android.dbflow.annotation.TypeConverter
public class CompanyConverter extends TypeConverter<String, UUID> {

  @Override
  public String getDBValue(UUID model) {
    return model.toString();
  }

  @Override
  public UUID getModelValue(String data) {
    return UUID.fromString(data);
  }
}
```
The second option could be using an auto-increment `int` keyin order to avoid the `generateUniqueIdentifier` method